Source/WebCore/Modules/webtransport/WebTransport.cpp

```diff
@@ -72,6 +73,19 @@ ExceptionOr<Ref<WebTransport>> WebTransport::create(ScriptExecutionContext& cont
     if (!dedicated && !options.serverCertificateHashes.isEmpty())
         return Exception { ExceptionCode::NotSupportedError };
 
+    HashSet<String> visited;
+    for (auto& protocol : options.protocols) {
+        if (!isValidHTTPToken(protocol))
+            return Exception { ExceptionCode::SyntaxError };
+
+        auto utf8 = protocol.utf8();
+        if (utf8.isNull() || utf8.length() > 512)
+            return Exception { ExceptionCode::SyntaxError };
+
+        if (!visited.add(protocol).isNewEntry)
```

Comment:
Could we call visited uniqueProtocols to make it more clear what is going on here?